low-level/sensors/impl/generic/raid_integrity_data.py

```diff
@@ -257,16 +260,13 @@ def _check_raid_state(self, device):
                         status = "success"
                         with open(self.output_file, 'w') as raid_file:
                             raid_file.write(RaidDataConfig.STATE_COMMAND_RESPONSE.value + "\n")
+                        self.alert_type = self.FAULT_RESOLVED
+                        self._alert_msg = "null"
+                        self._send_json_msg(self.alert_type, device, self._alert_msg)
```

Comment:
@shindeam Few questions with these changes now:
- Will we send the fault alert (for each of the three faults i.e. Missing: No Device Found, 
Fault: MISMATCH COUNT is found, as count does not match to the default '0' value. And
Fault: "'idle' state not found after max retries.") only once until there is a change in the state?
- After the fault has been resolved, will we send fault_resolved alert for each of the fault alerts above?

Reply:
@indrajitzagade Yes This is the expected behavior.